app/api/reviews/index.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express');
+
+const router = express.Router();
+
+const ProjectReviews = require('../../database/models/projectReviewsModel');
+
+router.post('/', async (req, res) => {
+  try {
+    const reviews = [req.body.likely_to_recommend, req.body.supporter_communication, req.body.provided_value];
+    let correctBool = true;
+    for (let i = 0; i < reviews.length; i += 1) {
+      if (!(reviews[i].isInteger() && reviews[i] >= 1 && reviews[i] <= 5)) {
+        correctBool = false;
+        break;
+      }
+    }
+    if (correctBool) {
+      const projectReview = await ProjectReviews.insert(req.body);
```

Comment:
This returns an array of inserted objects, even if you only pass in one parameter.
```javascript
const [projectReview] = await ProjectReviews.insert(req.body);
```